src/assets/translations/en/main.json

```diff
@@ -2867,14 +2868,11 @@
       "successSubtitle": "You have successfully staked %{amount} TCY",
       "goBack": "Go Back"
     },
-    "claimStatus": {
-      "pendingTitle": "Claim is pending",
-      "pendingSubtitle": "We are processing your claim for %{amount} TCY",
-      "successTitle": "Claim is successful",
-      "failedTitle": "Something went wrong",
-      "failedSubtitle": "Your claim has failed. Please try again.",
-      "successSubtitle": "You have successfully claimed %{amount} TCY",
-      "goBack": "Go Back"
```

Comment:
revert

Reply:
There was duplicate keys in here this should be removing the duplicate